### DISCUSSION TITLE
hosted version?

### DISCUSSION THREAD

**Discussion by Dieterbe** [mealie-recipes/mealie#3470] [General]
Hello!
is there any interst in offering a hosted version of mealie? (maybe for a small fee)
if not, does mealie support multi-tenancy? in other words, if i wanted to host mealie and invite the general public to use it, would that work well, such that people can't interfere with what other strangers are doing, etc? or is this more of a "personal" app? thanks!

**Comment by hay-kot** [comment]
There is not a hosted version of Mealie, I am working on a hosted recipe manager [Recipinned](https://recipinned.com/) that is a re-think/rewrite of Mealie built more for this use-case.

Mealie is _technically_ multi-tenant, however I abandoned providing a hosted version of Mealie because there are some large limitations I ran into testing it work larger workloads. It scales _really_ poorly and the sharing model doesn't work well for collaboration across groups, and there are still some security concerns hosting for untrusted users.

**Reply by Dieterbe** [reply]
OK. Thanks Hayden. I'm assuming recipinned is closed source ?

**Reply by hay-kot** [reply]
For now, yes. Once it’s stable I do plan to open source it. But it will likely be some time before that happens.
